data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -18,3 +18,24 @@ This script will update the sources to create ``cc-oci-runtime`` packages.
 
   * cc-oci-runtime.dsc
   * cc-oci-runtime.spec
+
+OBS
+---
+
+The script has two variables related to OBS so the CI can be used to push 
```

Comment:
Do not use passive voice:
The script has two OBS related variables. Using them, the CI can push changes to the [OBS website] (https://build.opensuse.org/).